src/gc/gc.cpp

```diff
@@ -16172,6 +16169,28 @@ void gc_heap::gc1()
             fire_pevents();
             pm_full_gc_init_or_clear();
 
+            // distribute gen0_must_clear_bricks to all heaps -
+            // if one heap encountered an interior pointer during this GC,
+            // the next GC might see one on another heap
+
+            // first, compute max
+            int max_gen0_must_clear_bricks = 0;
+            for (int i = 0; i < gc_heap::n_heaps; i++)
+            {
+                gc_heap* hp = gc_heap::g_heaps[i];
+                if (max_gen0_must_clear_bricks < hp->gen0_must_clear_bricks)
+                    max_gen0_must_clear_bricks = hp->gen0_must_clear_bricks;
```

Comment:
nit - could replace this with
max_gen0_must_clear_bricks = max (max_gen0_must_clear_bricks, hp->gen0_must_clear_bricks);

Reply:
Ok.